### PR TITLE
Admin panel

### PR DESCRIPTION
**Pull request by Maciorex**
Add admin actions for disciplines. Index, new and create.

### CODE REVIEW

File: app/controllers/disciplines_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    @discipline = Discipline.new
+  end
+
+  def create
+    @discipline = Discipline.new(discipline_params)
+    if @discipline.save
+      flash[:success] = "ŻEM DODOŁ DYSCYPLINA"
+      redirect_to disciplines_path
+    else
+      render 'new'
```
Comment (ozimeu) [inline app/controllers/disciplines_controller.rb]: maybe a flash message with errors here?

File: app/models/user.rb [hunk 2]
```diff
@@ -3,4 +3,6 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+
+
```
Comment (ozimeu) [inline app/models/user.rb]: empty lines not needed here

File: app/views/disciplines/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<h1 align="center"> List of disciplines </h1>
+
+<% @disciplines.each do |discipline| %>
+
```
Comment (ozimeu) [inline app/views/disciplines/index.html.erb]: we don't need this empty line here

File: app/views/disciplines/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    <div class="well well-lg">
+      <div class="discipline-name">
+        <%= discipline.name %>
+      </div>
+      <div class="discipline-description">
+        <%= discipline.description %>
+      </div>
+    </div>
+  </div>
+
```
Comment (ozimeu) [inline app/views/disciplines/index.html.erb]: we don't need this empty line here

File: app/views/disciplines/new.erb [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    <%= f.text_field :name, placeholder: "Discipline name" %>
+  </div>
+  <div class="form-group">
+    <%= f.label :description %>
+    <%= f.text_area :description, rows: 10, placeholder: "Discipline description" %>
+  </div>
+  <div class="form-group">
+    <%= f.submit "Add new discipline", class: "btn btn-primary btn-big" %>
+  </div>
+
```
Comment (ozimeu) [inline app/views/disciplines/new.erb]: we don't need this empty line here

File: app/models/discipline.rb [hunk 6]
```diff
@@ -0,0 +1,4 @@
+class Discipline < ApplicationRecord
+  validates :name, presence: true, uniqueness: true
+  validates :description, presence: true, length: {minimum: 10, maximum: 300}
```
Comment (ozimeu) [inline app/models/discipline.rb]: the `length` hash should have spaces inside, e.g. `{ min: 10 }`

File: app/controllers/disciplines_controller.rb (hunk 1 above)
Comment (Maciorex) [reply app/controllers/disciplines_controller.rb]: good idea, my master

File: app/models/user.rb (hunk 2 above)
Comment (Maciorex) [reply app/models/user.rb]: will correct

File: app/views/disciplines/index.html.erb (hunk 3 above)
Comment (Maciorex) [reply app/views/disciplines/index.html.erb]: vanish it!

File: app/views/disciplines/new.erb (hunk 5 above)
Comment (Maciorex) [reply app/views/disciplines/new.erb]: maaaaannnn

File: app/models/discipline.rb (hunk 6 above)
Comment (Maciorex) [reply app/models/discipline.rb]: correcto amigo

**Pull request merged**